lib/components/RecipientForm.jsx

```diff
@@ -24,6 +24,35 @@ const RecipientBalance = withFormProps(
       }
     })
     (class _RecipientBalance extends PureComponent {
+      componentDidMount () {
+        this.startSubscription()
+      }
+
+      componentDidUpdate () {
+        this.startSubscription()
```

Comment:
✂️ because you already have it in mount

Reply:
Done! I had copy-pasted from `RecipientForm`. I also removed it there.